hs_core/management/utils.py

```diff
@@ -161,6 +166,32 @@ def check_for_dangling_irods(echo_errors=True, log_errors=False, return_errors=F
     return errors
 
 
+class CheckJSONLD(object):
+    def __init__(self, short_id):
+        self.short_id = short_id
+
+    def test(self):
+        default_site = Site.objects.first()
+        validator_url = "https://search.google.com/structured-data/testing-tool/validate"
+        url = "https://" + default_site.domain + "/resource/" + self.short_id
+        cookies = {"NID": settings.GOOGLE_COOKIE_HASH}
+
+        response = post(validator_url, {"url": url}, cookies=cookies)
+
+        response_json = json.loads(response.text[4:])
+        if response_json.get("totalNumErrors") > 0:
+            for error in response_json.get("errors"):
+                if "includedInDataCatalog" not in error.get('args'):
+                    errors = response_json.get("errors")
+                    print("Error found on resource {}: {}".format(self.short_id, errors))
```

Comment:
Shouldn't you just log the ```error``` variable value here rather than logging the whole errors json response given the if condition you have to filter out ```includedInDataCatelog```?

Reply:
Thanks, Hong. the includedInDataCatalog thing is a pain, but Google may fix it soon. I will leave as-is for now.